src/main/kotlin/ch/hsr/dsl/dwrtc/websocket/Main.kt

```diff
@@ -3,13 +3,22 @@ package ch.hsr.dsl.dwrtc.websocket
 import ch.hsr.dsl.dwrtc.signaling.ClientService
 import ch.hsr.dsl.dwrtc.util.*
 import io.javalin.Javalin
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.server.ServerConnector
 
 /** Main start point. Fires up the Webserver. */
 fun main(args: Array<String>) {
     val app = Javalin.create()
             // .enableStaticFiles("src/main/resources/public/", Location.EXTERNAL)
-            .enableStaticFiles("/public")
-            .start(config[WEBSERVER_PORT])
+            .enableStaticFiles("/public").server {
+                Server().apply {
+                    connectors = arrayOf(ServerConnector(this).apply {
+                        this.host = config[WEBSERVER_IP]
```

Comment:
was passiert mit `null`?

Reply:
Beim port haben wir das bisher auch nicht gehandlet. Dort weiss ich aber dass er sich selbst einen sucht.

Hier nimmt er dann "localhost"

Reply:
Habe auch einen default in den default.properties gegeben. Würdest du das explizit handlen?